### PR TITLE
Add images for Longhorn v1.1.0 

### PR DESCRIPTION
**Pull request by shuo-wu**
Signed-off-by: Shuo Wu <shuo@rancher.com>

#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexographically) 
- [ ] Changes to scripting or CI config have been tested to the best of your ability

### CODE REVIEW

**Comment by yasker** [comment]
@cbron Do we need to provide images of all the Longhorn versions or just the latest version? I saw some other images have multiple versions.

**Comment by cbron** [comment]
Only the versions added in the latest chart.

File: images-list [hunk 1]
```diff
@@ -52,14 +52,16 @@ kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 0.1.151
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
 library/busybox rancher/library-busybox 1.31.1
 library/nginx rancher/library-nginx 1.19.2-alpine
-longhornio/csi-attacher rancher/longhornio-csi-attacher v2.0.0
```
Comment (cbron) [inline images-list:55]: Don’t remove the old versions.

**Review by cbron — CHANGES_REQUESTED** [review]

**Comment by shuo-wu** [comment]
BTW, the line `gcr.io/kubebuilder/kube-rbac-proxy rancher/kube-rbac-proxy v0.5.0` is out of order. But we won't correct it in this PR.

File: images-list [hunk 2]
```diff
@@ -53,13 +53,23 @@ kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
...
 library/nginx rancher/library-nginx 1.19.2-alpine
 longhornio/csi-attacher rancher/longhornio-csi-attacher v2.0.0
+longhornio/csi-attacher rancher/longhornio-csi-attacher v2.2.1-lh1
 longhornio/csi-node-driver-registrar rancher/longhornio-csi-node-driver-registrar v1.2.0
+longhornio/csi-node-driver-registrar rancher/longhornio-csi-node-driver-registrar v1.2.0-lh1
 longhornio/csi-provisioner rancher/longhornio-csi-provisioner v1.4.0
+longhornio/csi-provisioner rancher/longhornio-csi-provisioner v1.6.0-lh1
 longhornio/csi-resizer rancher/longhornio-csi-resizer v0.3.0
+longhornio/csi-resizer rancher/longhornio-csi-resizer v0.5.1-lh1
+longhornio/csi-snapshotter rancher/longhornio-csi-snapshotter v2.1.1-lh1
```
Comment (cbron) [inline images-list:63]: This repo doesn't exist, needs to be created first: https://hub.docker.com/r/rancher/longhornio-csi-snapshotter

File: images-list [hunk 3]
```diff
@@ -53,13 +53,23 @@ kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
...
 longhornio/longhorn-engine rancher/longhornio-longhorn-engine v1.0.2
+longhornio/longhorn-engine rancher/longhornio-longhorn-engine v1.1.0
 longhornio/longhorn-instance-manager rancher/longhornio-longhorn-instance-manager v1_20200514
+longhornio/longhorn-instance-manager rancher/longhornio-longhorn-instance-manager v1_20201216
 longhornio/longhorn-manager rancher/longhornio-longhorn-manager v1.0.2
+longhornio/longhorn-manager rancher/longhornio-longhorn-manager v1.1.0
+longhornio/longhorn-share-manager rancher/longhornio-longhorn-share-manager v1_20201204
```
Comment (cbron) [inline images-list:70]: Same here: https://hub.docker.com/r/rancher/longhornio-longhorn-share-manager

**Review by cbron — CHANGES_REQUESTED** [review]

File: images-list (hunk 2 above)
Comment (yasker) [reply images-list:63]: @cbron This is in the rancher org. Can someone in the Rancher team help to create those?

**Comment by yasker** [comment]
One suggestion: since we want the images to be sorted, we can add the validation of the order as a part of CI process, so any wrongly sorted image will be caught by the drone.

**Review by cbron — APPROVED** [review]

**Pull request merged**